index.js

```diff
@@ -63,15 +62,15 @@ module.exports = (uri, output, opts) => {
 		output = null;
 	}
 
-	opts = Object.assign({
-		encoding: null,
-		rejectUnauthorized: process.env.npm_config_strict_ssl !== 'false'
-	}, opts);
+	opts = {
+		rejectUnauthorized: process.env.npm_config_strict_ssl !== 'false',
+		...opts
+	};
 
 	const stream = got.stream(uri, opts);
 
 	const promise = pEvent(stream, 'response').then(res => {
-		const encoding = opts.encoding === null ? 'buffer' : opts.encoding;
+		const encoding = opts.encoding ? opts.encoding : 'buffer';
```

Comment:
Encoding option change due to change in `got` that detects any encoding value (incl null) as "oh you want a buffer".